src/components/ui/DateTimeInput/DateTimeInput.tsx

```diff
@@ -26,7 +26,10 @@ const DateTimeInput: React.FC<DateTimeInputProps> = ({
 
   useEffect(() => {
     if (value) {
-      setCurrentDate(new Date(value));
+      const newDate = new Date(value);
+      if (newDate.toString() !== "Invalid Date") {
+        setCurrentDate(newDate);
+      }
```

Comment:
I assume we'll throw validation errors when we implement the input forms?

Reply:
Yeah that can be done in the onChange function. I guess we need to verify if the date entered is in the past